user/enterprise/custom-queues.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Enterprise Customer Worker Queues
+layout: en_enterprise
+---
+
+Setting up custom worker queues on Travis CI Enterprise allows your team to designate a particular worker (or group of workers) for building specific repositories. This is especially useful for differentiating workers that have been customized specially.
+
+Custom queues require [enabling two feature flags](#Enable-Queues-on-the-Platform). Once these flags are set, you can define the configuration for your custom queues in the [Management Console settings](#Define-Custom-Queues-in-the-Management-Console), and [allocate workers to the new queues](#Define-Custom-Queues-Settings-on-The-Workers)
+
+<div id="toc"></div>
+
+## Enable Queues on the Platform
+
+To allow your Travis CI Enterprise platform instance to route jobs to customized queues, the `template_selection` and `multi_os` feature flags must be set. To do this, ssh into your platform server, then run `travis console`. Run the following command to enable the required feature flags: 
```

Comment:
Reword must be set into active voice? 

To allow.... set the ...

Reply:
Ah right. English likes active voice.